corgidrp/mean_combine.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
```

Comment:
I left it as a separate function because other things in the pipeline (https://collaboration.ipac.caltech.edu/pages/viewpage.action?spaceKey=romancoronagraph&title=Corgi-DRP+Implementation+Document#CorgiDRPImplementationDocument-II&Tmeancombine) use this (things that haven't been added to the DRP yet).

Reply:
What functions are those?